toothpick-kotlin/src/main/java/toothpick/kotlin/ToothpickExtensions.kt

```diff
@@ -0,0 +1,52 @@
+package toothpick.kotlin
+
+import toothpick.Scope
+import toothpick.Toothpick
+import toothpick.config.Binding
+import toothpick.config.Module
+import kotlin.reflect.KClass
+
+fun <T : Any> Module.bind(clazz: KClass<T>): Binding<T> {
+    return bind(clazz.java)
+}
+
+fun Scope.inject(toInject: Any) {
+    Toothpick.inject(toInject, this)
+}
+
+fun <T : Any> Scope.getInstance(clazz: KClass<T>, name: String? = null, annotationName: KClass<out Annotation>? = null): T {
```

Comment:
do we need the non reified ones ?

Reply:
When you want to inject a subclass into a super.

Reply:
Is there an example in your PR ?

Reply:
divide into two overloads to disambiguate name or annotationName, not both